src/state/megaStore.tsx

```diff
@@ -215,25 +217,17 @@ export const Provider: ParentComponent = (props) => {
                 // Get balance + price optimistically
                 const balance = await mutinyWallet.get_balance();
                 let price;
-                // only get price if balance is non-zero
-                if (
-                    balance.confirmed > 0 ||
-                    balance.unconfirmed > 0 ||
-                    balance.lightning > 0 ||
-                    balance.force_close > 0
-                ) {
```

Comment:
this was to improve our startup speed, might be worth keeping

Reply:
if we don't do it here, we have to put in logic to somehow fetch the price before the user's first receive. it's nice to speed up startup times but seems like it complicates the logic a lot